namespaces/live-1.cloud-platform.service.justice.gov.uk/dstest/resources/rds.tf

```diff
@@ -0,0 +1,33 @@
+variable "cluster_name" {
+}
+
+variable "cluster_state_bucket" {
+}
+
+module "dstest_rds" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=prevent_destroy"
+  cluster_name           = var.cluster_name
+  cluster_state_bucket   = var.cluster_state_bucket
+  team_name              = var.team_name
+  business-unit          = var.business-unit
+  application            = var.application
+  is-production          = var.is-production
+  environment-name       = var.environment-name
+  infrastructure-support = var.infrastructure-support
+  force_ssl              = "true"
```

Comment:
@digitalronin , can u remove the force_ssl as the db_parameter is arbitrary based on db engine.

Reply:
Done